flask_resty/testing.py

```diff
@@ -1,12 +1,36 @@
+from datetime import datetime, timedelta
+import re
+
+from dateutil import parser
+
 from collections import Mapping, Sequence
 import json
 
 from .compat import basestring
+from .utils import utc
 
 # -----------------------------------------------------------------------------
 
 UNDEFINED = object()
```

Comment:
if we had access to `mock` here this could/should be `sentinel.MISSING` or something... but we don't